Makefile

```diff
@@ -89,7 +90,7 @@ build-{{build_target}}-{{target_type}}:
 	mv -b {{target_dir}}/{{build_target}}/{{target_type}}/*.so {{lib_dir}}/{{build_target}}
```

Comment:
It seems that this does not work currently with files generated by `project-utils`.

Reply:
yep, I believe @karroffel should adjust the `project-utils` for this...

Reply:
I'm not 100% sure what's the best way to deal with this in `project-utils`. I could add a flag that disables generation of the `gdnlib` file so that the user *has* to provide it? I'm not sure how to make it customisable enough *and* convenient by default. If you have suggestions I'll try to implement them.

Reply:
@karroffel lets discuss it on discord

Reply:
lets discuss it on discord